golem-worker-service-base/src/gateway_api_definition/http/api_definition.rs

```diff
@@ -219,6 +217,38 @@ impl TryFrom<grpc_apidefinition::ApiDefinition>
     }
 }
 
+impl TryFrom<HttpApiDefinition> for grpc_apidefinition::ApiDefinition {
```

Comment:
These were moved next to the types they belong to. I think some leftover from some cleanup